sopel/trigger.py

```diff
@@ -175,6 +180,10 @@ def __init__(self, own_nick, line):
             # Account is the second arg `...JOIN #Sopel account :realname`
             self.tags['account'] = self.args[1]
 
+        # get plain text message
+        if self.args:
+            self.plain = formatting.plain(self.args[-1])
```

Comment:
Why not use `self.text`? It's already initialized to either the last arg, or the part of the line after <tt>'&nbsp;:'</tt> if present.

```suggestion
        # get plain text message
        if self.text:
            self.plain = formatting.plain(self.text)
```

Reply:
Because `self.text`, in the case of a CTCP, will contain `\x01ACTION text\x01` which is not what I thought we wanted.

Reply:
Hmm, arguably `self.text` shouldn't contain the CTCP bytes either, but that's a separate conversation.